src/API/Terra.ts

```diff
@@ -128,8 +128,8 @@ export default class Terra {
    * @return {Promise<TerraUserResponse>} A promise of type User
    *
    */
-  getUser(userID: string): Promise<TerraUserResponse> {
-    return GetUser(this.devID, this.apiKey, userID);
+  getUser(userID: string, referenceID: string): Promise<TerraUserResponse> {
```

Comment:
use a struct {userid, refid} so they can be optionally passed otherwise you're forcing both